### PR TITLE
Create react-list-of-goods-hooks

### PR DESCRIPTION
**Pull request by Pavlo-Khashchevskyi**
[DEMO LINK](https://Pavlo-Khashchevskyi.github.io/react_list-of-goods/)

### CODE REVIEW

**Comment by Pavlo-Khashchevskyi** [comment]
> [DEMO LINK](https://Pavlo-Khashchevskyi.github.io/react_list-of-goods-hooks/)

File: src/App.tsx [hunk 1]
```diff
@@ -14,11 +16,114 @@ const goodsFromServer: string[] = [
...
+    setRevers(false);
+    setTypeSort('none');
+    setFildSize(goodsFromServer.length);
+  }
+
+  return (
+    <div className="App bg-info">
+      <div className="App-wrap">
+        {!start && (
+          <button className="btn bg-primary btn-my" type="button" onClick={() => setStart(true)}>Start</button>
```
Comment (vitaliy-martynyuk) [inline src/App.tsx:35]:
```suggestion
          <button
            className="btn bg-primary btn-my"
            type="button"
            onClick={() => setStart(true)}
          >
            Start
          </button>
```

**Review by vitaliy-martynyuk — CHANGES_REQUESTED** [review]
you forgot to add `GoodsList` related files

File: src/App.tsx (hunk 1 above)
Comment (Pavlo-Khashchevskyi) [reply src/App.tsx]: I corrected the errors, but now I can't request a check again

**Review by varseniuk — COMMENTED** [review]
Закритий ПР. Не можу порев'ювити

**Pull request closed without merging**